lib/ex_unit/lib/ex_unit.ex

```diff
@@ -312,6 +314,19 @@ defmodule ExUnit do
     ExUnit.Runner.run(config, nil)
   end
 
+  @doc """
+  Allows users to configure functions to run after the test suite has finished.
+  Functions passed in will be executed after all tests have finished running. If
+  `after_suite/1` is called multiple times, the functions will be executed in
+  reverse order (i.e. the last function given will be executed first).
+  """
+  @spec after_suite(fun) :: :ok
+  def after_suite(function) do
```

Comment:
Maybe use a guard here `is_function(function, 0)`?

Reply:
We generally prefer `is_function(function)` as it leads to better error messages when invoking with the wrong arity later on.

Reply:
Funny enough I was thinking the same thing last night as I was lying in bed! I'll definitely add that.